src/server/pfs/server/server.go

```diff
@@ -40,10 +42,21 @@ func NewObjBlockAPIServer(dir string, cacheBytes int64, objClient obj.Client) (p
 	return newObjBlockAPIServer(dir, cacheBytes, objClient)
 }
 
-// NewBlockAPIServer creates a BlockAPIServer using the credentials it finds in
-// the environment
+// NewBlockAPIServer creates a BlockAPIServer using the credentials
+// it finds in the environment
 func NewBlockAPIServer(dir string, cacheBytes int64, backend string) (pfsclient.BlockAPIServer, error) {
+	log.Info("Initializing new blockAPIServer", dir, cacheBytes, backend)
```

Comment:
Did you mean to leave this in? It doesn't seem like a particularly important thing to inform the user of, you'll basically just see this once on startup. Also log messages should use lion right now to match the rest of our code.